test/e2e/specs/editor/blocks/spacer.spec.js

```diff
@@ -48,4 +48,27 @@ test.describe( 'Spacer', () => {
 			)
 		).toBeFocused();
 	} );
+
+	test( 'should work in theme without spacing units support', async ( {
+		editor,
+		page,
+	} ) => {
+		// Mock the theme.json data to simulate a theme without spacing units
```

Comment:
```suggestion
		await page.waitForFunction( () => window?.wp?.data );

		// Mock the theme.json data to simulate a theme without spacing units
```

Ensures `window.wp.data` is available before running the settings update.